src/components/Form/Form.react.js

```diff
@@ -45,14 +46,20 @@ function Form({
   action,
   method,
   onSubmit,
+  autocomplete,
 }: Props): React.Node {
+  // default for autocomplete is "off" if not passed in explicitly
+  if (!autocomplete) {
```

Comment:
Maybe use `defaultProps` instead? see [Flow's defaultProps](https://flow.org/en/docs/react/components/#toc-using-default-props-for-functional-components)

```jsx
defaultProps = {
    autocomplete: "off";
}
```

Reply:
The `Table` component uses it as a out-of-function definition like so:

```jsx
Table.defaultProps = {
  cards: false,
  striped: false,
  responsive: false,
};
```